python/objToJSON.c

```diff
@@ -263,7 +263,9 @@ static int Dict_iterNext(JSOBJ obj, JSONTypeContext *tc)
 
   if (PyUnicode_Check(GET_TC(tc)->itemName))
   {
+    itemNameTmp = GET_TC(tc)->itemName;
     GET_TC(tc)->itemName = PyUnicode_AsUTF8String (GET_TC(tc)->itemName);
```

Comment:
Use `itemNameTmp` on he right-hand side? 

Also remove `#if PY_MAJOR_VERSION >= 3` around its declaration on line 242.

Reply:
`itemNameTmp` is used similarly in line 279-281, so I thought to prefer to use the same structure here for consistency though I agree `itemNameTmp` would be better.